src/main/java/Printer.java

```diff
@@ -1,10 +1,15 @@
 public class Printer {
-    private final String bar = "____________________________________________________________";
```

Comment:
could this be better if it was a final String named BAR instead of the change you made?